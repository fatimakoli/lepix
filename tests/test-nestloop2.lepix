fun main():int {
	var i:int = 0;
	var x:int = 7;

	for(i;i < 10;i = i + 1) {
		if(x > 3) {
			print(42);
			if(x < 4) {
				print(21);
			}
			else {
				print(60);
			}
		}
	}
	return 0;
}