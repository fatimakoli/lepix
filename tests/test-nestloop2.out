42
60
42
60
42
60
42
60
42
60
42
60
42
60
42
60
42
60
42
60
